infoschema/cluster_tables_test.go

```diff
@@ -701,7 +701,7 @@ select * from t1;
 		checkFn(quota)
 	}
 	for i := 0; i < 100; i++ {
-		quota := rand.Int()%10240 + 1
+		quota := rand.Int()%8192 + 1
```

Comment:
unrelated change?

Reply:
I found `10240` maybe too large and won't cause oom in my local test sometime, so I change it to a smaller value.